plugins/streaming/impls/jms/src/main/java/org/pentaho/di/trans/step/jms/JmsDelegate.java

```diff
@@ -73,6 +73,26 @@
 
   @Injection ( name = "DESTINATION_FIELD_NAME" ) public String destinationField = "destination";
 
+  @Injection ( name = "SSL_ENABLED" ) public boolean sslEnabled = false;
+
+  @Injection ( name = "SSL_KEYSTORE_PATH" ) public String sslKeystorePath = "";
+
+  @Injection ( name = "SSL_KEYSTORE_TYPE" ) public String sslKeystoreType = "";
+
+  @Injection ( name = "SSL_KEYSTORE_PASSWORD" ) public String sslKeystorePassword = "";
+
+  @Injection ( name = "SSL_TRUSTSTORE_PATH" ) public String sslTruststorePath = "";
+
+  @Injection ( name = "SSL_TRUSTSTORE_TYPE" ) public String sslTruststoreType = "";
+
+  @Injection ( name = "SSL_TRUSTSTORE_PASSWORD" ) public String sslTruststorePassword = "";
+
+  @Injection ( name = "SSL_CONTEXT_ALGORITHM" ) public String sslContextAlgorithm = "";
+
+  @Injection ( name = "IBM_SSL_CIPHERSUITE" ) public String ibmSslCipherSuite = "";
+
+  @Injection ( name = "IBM_SSL_FIPSREQUIRED" ) public String ibmSslFipsRequired = "";
+
```

Comment:
Hey Peter- sorry, I shoulda noticed this before.  We probably want to group the SSL props, and mark the passwords as @Sensitive.  Similar to what we did with MQTT:  https://github.com/mkambol/pentaho-kettle/blob/9970bb974d9b268a666042c7976193f05c36306e/plugins/streaming/impls/mqtt/src/main/java/org/pentaho/di/trans/step/mqtt/MQTTConsumerMeta.java#L107-L108